spec/unit/puppet/provider/network_vlan/cisco_nexus_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'spec_helper'
+require 'cisco_node_utils'
+
+ensure_module_defined('Puppet::Provider::NetworkVlan')
+require 'puppet/provider/network_vlan/cisco_nexus'
+
+RSpec.describe Puppet::Provider::NetworkVlan::CiscoNexus do
+  let(:provider) { described_class.new }
+  let(:context) { instance_double('Puppet::ResourceApi::BaseContext', 'context') }
+  let(:device) { instance_double('Puppet::Util::NetworkDevice::Nexus::Device', 'device') }
+  let(:networkvlan) { instance_double('Cisco::NetworkVlan', 'networkvlan') }
+
+  let(:vlan) { instance_double('Cisco::Vlan', 'vlan') }
+  let(:vlan2) { instance_double('Cisco::Vlan', 'vlan2') }
+  let(:vlans) do
+    {
+      '1' => vlan,
+      '2' => vlan2,
+    }
+  end
+  let(:should_values) do
+    {
+      id: '42',
+      vlan_name: 'vlan_42',
+      ensure: 'present',
+      shutdown: true,
+    }
+  end
+  let(:changes) do
+    {
+      '42' =>
+            {
+              is: is_values,
+              should: should_values
+            }
+    }
+  end
+
+  before(:each) do
+    allow(context).to receive(:device).and_return(device)
+    allow(device).to receive(:facts).and_return('operatingsystem' => 'nexus')
+  end
+
+  describe '#get' do
+    it 'processes resources' do
+      allow(Cisco::Vlan).to receive(:vlans).and_return(vlans)
+
+      expect(vlan).to receive(:vlan_name).and_return('test1')
+      expect(vlan).to receive(:shutdown).and_return(false)
+      expect(vlan2).to receive(:vlan_name).and_return('test2')
+      expect(vlan2).to receive(:shutdown).and_return(true)
+
+      expect(provider.get(context)).to eq [
+        {
+          id: '1',
+          ensure: 'present',
+          shutdown: false,
+          vlan_name: 'test1',
+        },
+        {
+          id: '2',
+          ensure: 'present',
+          shutdown: true,
+          vlan_name: 'test2',
+        },
+      ]
+    end
+  end
+
+  describe '#set' do
+    context 'create' do
+      let(:is_values) do
```

Comment:
Same with the `is_values` here and in the `update` context, could remove both of them.

Reply:
Removed.